models/models.go

```diff
@@ -69,6 +69,16 @@ type StepToolkitModel struct {
 	Go   *GoStepToolkitModel   `json:"go,omitempty" yaml:"go,omitempty"`
 }
 
+// metaModel contains arrays with addons' identifiers that are required or optional to be connected for the step to function correctly
+type metaModel struct {
```

Comment:
I think this model should be simply `map[string]interface{}` so it would be an universal one, so we won't need to modify this field each time a new meta info need to be stored

Reply:
agree we will not use this property of steps, we just need to keep it when we are parsing step and bitrise ymls.